sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -13,6 +13,7 @@
 			<![CDATA[
 New features:
 - Added support for serverless databases.
```

Comment:
Line 15 is release notes for 1.31.0-preview, this was already published (see https://www.nuget.org/packages/Microsoft.Azure.Management.Sql). So you should update version to 1.32 (here and in AssemblyInfo.cs) and wipe out the old release notes.